src/models/Order.js

```diff
@@ -69,10 +72,14 @@ const orderSchema = new mongoose.Schema(
 // Create a function to update order statuses
 const updateOrderStatus = async () => {
   try {
-    // Get the current date and subtract 1 minute
-    const oneMinutesAgo = new Date(Date.now() - 1 * 60 * 1000);
+    let isDevelopment = process.env.NODE_ENV !== 'production';
+    const timeDuration = isDevelopment
+      ? parseInt(process.env.DEV_TIME_DURATION)
+      : parseInt(process.env.PROD_TIME_DURATION);
+    // Get the current date and subtract __x   minutes
+    const oneMinutesAgo = new Date(Date.now() - timeDuration);
```

Comment:
Since this is no longer always a minute ago, maybe rename the variable something more general that makes sense in both cases: `expiryTimeSinceCreation`

Reply:
@akosasante sure!! I was thinking about it too